src/views/collection-view/helpers.ts

```diff
@@ -9,6 +9,8 @@ import { splitProductFromFilename } from 'views/tutorial-view/utils'
  * reference a different product context
  */
 
+const BETA_PRODUCTS = ['vault', 'waypoint']
```

Comment:
Perhaps this is something we lift up, presuming other areas of the codebase may need this conditional treatment of products. Or does this already exist somewhere??

Reply:
Agree that this should probably be higher up and I don't believe it exists yet, I'd like to try handling these with feature flags and _potentially_ adding something like `isBetaProduct` to the `CurrentProduct` context. I plan on toying around with this a little in primary nav work! What do you think of these ideas? At the very least, we can do the latter without feature flags and just hardcode the array of beta products in `CurrentProduct` context.

Reply:
I think that's a good idea! Although in this case the collection links wouldn't be referencing the `CurrentProduct`....like they could be called within the vault product context (if that were current) and be referencing _other_ product collections, if that makes sense.

Reply:
Ah that's a great point! I keep forgetting how that's possible in Learn content. 😅

We can probably create a new hook/util instead then that accepts a product slug as an argument, and checks against an array of beta products (like you have here) or another option is that we use a beta feature flag for each product. I think either would work! The latter would require a config change as more products are added, the former would require code changes. Both require rebuilds I believe, so I think they have the same tradeoffs!